blog-article.html

```diff
@@ -104,48 +107,79 @@ <h2 class="article-heading">NFT Avatar Generators</h2>
             <ul class="social-list">
                 <li class="social-icon">
                     <a href="#" class="social-link">
-                        <img src="#" alt="Discord">
+                        <img src="images\icons\discord.svg" alt="Discord" width="20" height="14.78">
                     </a>
                 </li>
                 <li class="social-icon">
                     <a href="#" class="social-link">
-                        <img src="#" alt="Twitter">
+                        <img src="images\icons\twitter.svg" alt="Twitter" width="20" height="20">
                     </a>
                 </li>
                 <li class="social-icon">
                     <a href="#" class="social-link">
-                        <img src="#" alt="YouTube">
+                        <img src="images\icons\youtube.svg" alt="YouTube" width="20" height="14.81">
                     </a>
                 </li>
             </ul>
         </div>
         <div class="article-browsing">
             <a href="#" class="previous-article">
                 <article class="article-figure">
-                    <img src="#" alt="playing card">
-                    <h3 class="article-text">PREVIOUS</h3>
+                    <img src="images\nft\playing-card.webp" alt="playing card" width="80" height="80">
+                    <h3 class="article-text sub-heading">PREVIOUS</h3>
                     <p class="sub-text">Senior Apple employee alleges sexism at</p>
                 </article>
             </a>
             <a href="#" class="next-article">
                 <article class="article-figure">
-                    <img src="#" alt="cat">
-                    <h3 class="article-text">NEXT</h3>
+                    <img src="Images\nft\cat.webp" alt="cat" width="80" height="80">
+                    <h3 class="article-text sub-heading"><span>NEXT</span></h3>
                     <p class="sub-text">Senior Apple employee alleges sexism at</p>
                 </article>
             </a>
         </div>
         <aside class="sidebar">
-            <form class="search-bar" action="/search" method="get">
-                <input class="search-window" type="text" name="query" placeholder="Search NFT">
-                <input class="search-image" type="image" src="#" alt="search-icon">
-            </form>
+            <form class="search-bar" method="get">
+                <input class="search-input" type="text" name="search" placeholder="Search NFT">
+                <label for="search"></label>
+                <button class="search-btn" type="button"><svg width="18" height="18" viewBox="0 0 18 18" fill="none"

```

Comment:
When a button is intended to submit a form, it should be of type submit. This is the default type for buttons within forms, and it ensures that the form data is sent to the server when the button is clicked. Using the correct button type is crucial for form functionality. [Learn more about button types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type).